package.json

```diff
@@ -62,9 +63,5 @@
     "svgo": "2.8.0",
     "updates": "13.1.2"
   },
-  "browserslist": [
-    "defaults",
-    "not ie > 0",
-    "not ie_mob > 0"
-  ]
+  "browserslist": ["defaults", "not ie > 0", "not ie_mob > 0"]
```

Comment:
Unrelated

Reply:
https://github.com/go-gitea/gitea/pull/20458/commits/559cf0da2b997e3e5e86ef327f6e648fdb1d97ad IDE is pulling a funny over me by formatting this as a JSON file.